packages/mongo-nav/src/types.ts

```diff
@@ -176,8 +176,8 @@ export interface AccountInterface {
 export interface ProjectInterface {
   projectId: string;
   projectName: string;
-  planType: PlanType;
-  orgId: string;
+  planType?: PlanType;
+  orgId?: string;
```

Comment:
[commentary] these are marked as optional because the serverside endpoint used for admin filtering does not return a planType or orgId as part of the `GroupInfoView` class that is prepared and returned.